playwright-specflow-xunit/Steps/Hooks.cs

```diff
@@ -0,0 +1,79 @@
+using BoDi;
+using Microsoft.Playwright;
+using Newtonsoft.Json;
+
+namespace SpecFlowPlaywrightXUnitExample.Steps
+{
+    [Binding]
+    internal class Hooks
+    {
+        public IBrowser browser;
+        public IBrowserContext context;
+        public IPage page;
+        public IPlaywright playwright;
+        private Dictionary<string, object>? _capabilities = new Dictionary<string, object>();
+        private string gdpUrl = "";
+        private readonly IObjectContainer _objectContainer;
+        private readonly ScenarioContext _scenarioContext;
+
+        public Hooks(IObjectContainer objectContainer, ScenarioContext scenarioContext)
+        {
+            _objectContainer = objectContainer;
+            _scenarioContext = scenarioContext;
+        }
+        [BeforeScenario()]
+        public async Task CreateCapabilities()
+        {
+            string user, accessKey;
+            user = Environment.GetEnvironmentVariable("LT_USERNAME");
+            accessKey = Environment.GetEnvironmentVariable("LT_ACCESS_KEY");
+            string tags = string.Join(",", _scenarioContext.ScenarioInfo.Tags);
+            if(tags.Contains("chrome")) {
+                _capabilities.Add("browserName", "Chrome");
+
+            } else if(tags.Contains("edge")) {
+                _capabilities.Add("browserName", "MicrosoftEdge");
+
+            } else {
+                _capabilities.Add("browserName", "Firefox");
```

Comment:
`pw-firefox` has to be used as browserName for launching Firefox browser.
Can we get the exact browserName string from the tags and pass it here, rather than having these if else conditions?